uc-berkeley-ieee.ghost.2020-05-18-22-42-22.json

```diff
@@ -0,0 +1 @@
d122c","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-26T20:13:27.000Z"},{"id":"5e56d1720f6e325a8d9d122e","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-26T20:13:38.000Z"},{"id":"5e56d1770f6e325a8d9d1230","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-26T20:13:43.000Z"},{"id":"5e56d1970f6e325a8d9d1232","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-26T20:14:15.000Z"},{"id":"5e56d1bc0f6e325a8d9d1234","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-26T20:14:52.000Z"},{"id":"5e5719370f6e325a8d9d1236","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:19:51.000Z"},{"id":"5e5719480f6e325a8d9d1238","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:20:08.000Z"},{"id":"5e5719b70f6e325a8d9d123a","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:21:59.000Z"},{"id":"5e5719c80f6e325a8d9d123c","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:22:16.000Z"},{"id":"5e571a0c0f6e325a8d9d123f","resource_id":"5e571a0c0f6e325a8d9d123d","resource_type":"tag","actor_id":"5e56cde70f6e325a8d9d121e","actor_type":"user","event":"added","context":null,"created_at":"2020-02-27T01:23:24.000Z"},{"id":"5e571a0c0f6e325a8d9d1240","resource_id":"5e44abde0f6e325a8d9d0981","resource_type":"post","actor_id":"5e56cde70f6e325a8d9d121e","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:23:24.000Z"},{"id":"5e571a180f6e325a8d9d1242","resource_id":"5e41e9140f6e325a8d9d0961","resource_type":"post","actor_id":"5e56cde70f6e325a8d9d121e","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:23:36.000Z"},{"id":"5e571a330f6e325a8d9d1244","resource_id":"5e56d1250f6e325a8d9d1222","resource_type":"post","actor_id":"5e40f6300f6e325a8d9d0945","actor_type":"user","event":"edited","context":null,"created_at":"2020-02-27T01:24:03.000Z"},{"id":"5e571a5c0f6e325a8d9d1246","resource_id":"5e56d1250f6e325a
```

Comment:
This seems to be sensitive information from your personal instance of Ghost, I would probably `git rm` this so it doesn't show up in the branch.

Reply:
Should a force push be used here? It seems that this JSON file is from our active Ghost website.

Reply:
Oh yeah this is the content of our website. I just thought I would include it.. idk why lol. I can delete it.